In [1]:
name = "Ana"
hours = 40
hourly_wage = 16
weekly_pay = hours * hourly_wage
print(f"{name} earns {weekly_pay} per week.")

Ana earns 640 per week.


Need to look up how to get one decimal place added to the pay to become 640.0.^

In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]
high = 0
for s in temps:
    if s > 65:
        high = high + 1
print(high)

4


In [3]:
def to_celcius(f):
    return (f-32) * 5 / 9

print(round(to_celcius(0), 1))

-17.8


This is not the answer given on the homework but I believe that 0 Fahrenheit does equal -17.8. I checked by entering 32F and got 0C, and adding 212F and got 100C. The formula is correct.

In [4]:
book = {"title": "Empire of the Sun", "year": 1987, "rating": 10}
for key, value in book.items():
    print(key, ":", value)

title : Empire of the Sun
year : 1987
rating : 10


In [5]:
import pandas as pd
df = pd.read_csv('chickwts.csv')
df.head()

,Unnamed: 0,weight,feed
0,1,179,horsebean
1,2,160,horsebean
2,3,136,horsebean
3,4,227,horsebean
4,5,217,horsebean


In [6]:
df.shape

(71, 3)

In [7]:
df.describe()

,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


The describe function tells us that of the 71 chicks measured, the average weight was 261.3 (assume grams). The minimum weight was 108 grams and the maximum 423 grams. There is no information about the feed in this summary so we lose a lot of information. Describe only describes a few statistics of the numerical fields.